Examples/Physics_applications/plasma_acceleration/PICMI_inputs_plasma_acceleration_1d.py

```diff
@@ -0,0 +1,71 @@
+from pywarpx import picmi
+#from warp import picmi
+
+constants = picmi.constants
+
+nz = 64
+
+zmin = -200.e-6
+zmax = +200.e-6
+
+moving_window_velocity = [0., 0., constants.c]
+
+number_per_cell_each_dim = [2, 1, 1]
```

Comment:
Does PICMI always expect a len 3 list here?

particles: maybe a few more makes sense per cell, e.g., `10`?

Reply:
No, just one number in the list is enough.